libnavui-androidauto/src/main/java/com/mapbox/androidauto/MapboxCarApp.kt

```diff
@@ -44,11 +46,25 @@ object MapboxCarApp {
 
     /**
      * Setup android auto from your [Application.onCreate]
-     *
-     * @param application used to detect when activities are foregrounded
      */
-    fun setup(application: Application) {
-        SharedApp.setup(application)
+    fun setup() {
+        MapboxNavigationApp.registerObserver(sharedServices)
         MapboxNavigationApp.registerObserver(CarAppLocationImpl())
     }
+
+    // TODO can be replaced with `SharedApp` once backwards compatibility is fixed
+    //    https://github.com/mapbox/mapbox-navigation-android/pull/6303
+    private val sharedServices = object : MapboxNavigationObserver {
```

Comment:
After some testing with @Zayankovsky we're finding 2 instances of `MapboxAudioGuidanceImpl`.

User experience may be find, but technically this is not what we want. Am closing this approach.

I'm not finding another short term compatibility fix, so we need to bump AA to `v2.8.0-beta.3`

Reply:
@kmadsen do we need the version `2.8.0-beta.3.1` or might it be done separately? (SDK 2.8.0-beta.3 is already live)